policyengine_us/variables/gov/states/mo/tax/income/adjusted_gross_income/mo_adjusted_gross_income.py

```diff
@@ -15,5 +15,38 @@ class mo_adjusted_gross_income(Variable):
 
     def formula(person, period, parameters):
         gross_income = person("irs_gross_income", period)
+        # subtract federal above-the-line deductions (ALDs) by person
+        # ... subtract some ALDs explicitly by person
+        personal_alds = [
+            "self_employment_tax_ald_person",
+            "self_employed_health_insurance_ald_person",
+            "self_employed_pension_contribution_ald_person",
+        ]
+        tax_unit = person.tax_unit
+        ind_total_personal_alds = add(person, period, personal_alds)
+        unit_total_personal_alds = add(tax_unit, period, personal_alds)
+        fed_agi = gross_income - ind_total_personal_alds
```

Comment:
To avoid overwriting `fed_agi`, and since we don't need it until we can reduce by both types of ALDs.
```suggestion
```

Reply:
Done